tests/computation/test_post_analysis.py

```diff
@@ -0,0 +1,97 @@
+import json
+import pytest
+from oasislmf.manager import OasisManager
+
+
+def write_simple_post_analysis_module(module_path, class_name='PostAnalysis'):
+    with open(module_path, 'w') as f:
+        f.write(f'''
+from pathlib import Path
```

Comment:
Is there a reason these modules are imported inside of this test, rather than at the top of the file?

Reply:
Ah, I see it is used due to the way the module is imported.